PatientRegistrationApp/src/components/HouseholdEmergencyContact.vue

```diff
@@ -63,43 +62,42 @@ export default {
     };
   },
   methods: {
-    /*sendHouseholdEmergencyContactInfoToHouseholdReviewPage()
+    sendHouseholdEmergencyContactInfoToReviewPage()
     {
-      const emergencyContactPayload = {
-        emergencyContactFamilyName: this.householdEmergencyContactFamilyName,
-        emergencyContactGivenName: this.householdEmergencyContactGivenName,
-        emergencyContactPhoneNumber: this.householdEmergencyContactPhoneNumber,
-        emergencyContactPhoneNumberType: this.householdEmergencyContactPhoneNumberType
+      const householdEmergencyContactPayload = {
+        householdEmergencyContactFamilyName: this.householdEmergencyContactFamilyName,
+        householdEmergencyContactGivenName: this.householdEmergencyContactGivenName,
+        householdEmergencyContactPhoneNumber: this.householdEmergencyContactPhoneNumber,
+        householdEmergencyContactPhoneNumberType: this.householdEmergencyContactPhoneNumberType
       }
-      EventBus.$emit('DATA_EMERGENCY_CONTACT_INFO_PUBLISHED', emergencyContactPayload)
-    },*/
+      EventBus.$emit('DATA_HOUSEHOLD_EMERGENCY_CONTACT_INFO_PUBLISHED', householdEmergencyContactPayload)
+    },
     verifyFormContents()
     {
-		//add logic to check form contents
-		var valid = true
-		var message = "Woops! You need to enter the following fields:"
-		
-		if(this.householdEmergencyContactFamilyName == "")  {
-			message += " *Last Name"
-			valid = false
-		}
-		
-		if(this.householdEmergencyContactGivenName == "") {
-			message += " *First Name"
-			valid = false
-		}
-		
-		
-		if (this.householdEmergencyContactPhoneNumber == "") {
-			message += " *Phone Number"
-			valid = false
-		}
-		
-		if (valid == false) {
-				alert(message)
-				return false
-			}
-      //this.sendHouseholdEmergencyContactInfoToReviewPage();
+      //add logic to check form contents
+      var valid = true
+      var message = "Woops! You need to enter the following fields:"
+      
+      if(this.householdEmergencyContactFamilyName == "")  {
+        message += " *Last Name"
+        valid = false
+      }
+      
+      if(this.householdEmergencyContactGivenName == "") {
+        message += " *First Name"
+        valid = false
+      }
+      
+      if (this.householdEmergencyContactPhoneNumber == "") {
```

Comment:
Same as above for nesting this and the `valid==false` conditional

Reply:
This wasn't part of my change - if GitHub says it is, perhaps it was due to a manual merge or rearrangement of lines. Create a separate card for this?

However, I think the logic is fine - if any one of the conditions above is false, the overall validation of the page is false (but we want to know exactly which conditions fail so that we can tell the user what to correct)

Reply:
I think a separate card would suffice. Probably doing too many things on a given branch/card that are out of scope. 

If you think the logic is good, good with me. Just wanted to check!